reporter.go

```diff
@@ -104,13 +104,13 @@ func (r *KubernetesReporter) logPods(dirName string) {
 		if !r.namespaceToLog(pod.Namespace) {
 			continue
 		}
-		f, err := logFileFor(r.reportPath, dirName, pod.Namespace+"-pods_specs")
+		f, err := logFileFor(r.reportPath, dirName, pod.Namespace+"_"+pod.Name+"-pods_specs")
```

Comment:
nit: is it intentional to have namespace_podname-pod_specs?
If not, can we have all _ or all - ?

Reply:
moving all to `_`, as namespaces and pod names might contain `-`